### DISCUSSION TITLE
Silicone dog tag engraving

### DISCUSSION THREAD

**Discussion by targetsys** [arkypita/LaserGRBL#1715] [Q&A]
Hi
I am trying to engrave silicone dog tags with Atomstack A5
It should look like this one:
![silidog](https://user-images.githubusercontent.com/97455129/148785530-99ba9e6b-78a4-4831-854f-e1a926651a6a.jpg)

I get bad results' s
![test1](https://user-images.githubusercontent.com/97455129/148785451-d478369c-6e66-4fa3-8229-e7343c74f335.jpg)

any hint how I have to configure the GRBL to make it deeper.

Thanks

**Comment by Tribo-2dot0** [comment]
Deeper, you'll probably have to slow down the engrave speed or increase the laser power.

**Reply by targetsys** [reply]
I tried speed 800 and it still like this
can you suggest GRBL parameters that I can start with. for my tests.
Thanks

**Reply by gmmanonymus111** [reply]
I'd go with 250mm/m (or even 200), with air assist. But this is a case you have to experiment with to get the best results.
